src/storage.rs

```diff
@@ -42,6 +42,12 @@ pub trait StorageExt: Storage {
     /// Append the new entries to storage
     fn append(&self, ents: &[Entry]) -> Result<(), Error>;
 
+    /// Get the index of the raft entry that corresponds to the last block that was committed
```

Comment:
No reason to mention blocks here, since the StorageExt trait isn't operating on them and it leaks how we're using it.